scrap_reason_code/views/stock_scrap_views.xml

```diff
@@ -17,7 +21,11 @@
         <field name="inherit_id" ref="stock.stock_scrap_tree_view" />
         <field name="arch" type="xml">
             <xpath expr="//field[@name='product_uom_id']" position='after'>
-                <field name="reason_code_id" />
+                <field
+                    name="reason_code_id"
+                    required="1"
+                    domain="['|', ('company_id', '=', company_id), ('company_id', '=', False)]"
```

Comment:
Don't do this here, add multi company security rules for the reason code model.

Reply:
correct me if I am wrong. In V13 you can have multiple companies active at the same time. so reason can not be from the current company of user but it has to be the company from the stock scrap/Stock picking is in

Reply:
Same as in the other PR, you are right, and you can keep it, but multicompany rules can still be added (and I think they should) and even a constraint to check that no inconsistency is entered.

Reply:
Done